### PR TITLE
Bootstrapify identifications by login

### PR DESCRIPTION
**Pull request by seanclifford**
Made "Identifications by login" page use the bootstrap subnav header as part of issue #958.

I started by just changing the header, but found the default layout not wide enough to hold the bootstrap subnav header without wrapping, so figured I'd try applying bootstrap styles to the whole page, while trying to keep the original design.
I ended up having to apply a few css hacks that I'm not too happy with, but thought I'd put it out there and see what you think...

Summary of changes:
- Applied bootstrap layout, and used bootstrap grid column styles
- Updated `by_login.html` and `_identification_with_observation.html` to use Haml
- Removed some unused dead code around activity streams because it was referencing `_identification_with_observation`
- Fixed bootstrap subnav so the IDs tab was active when displaying identifications by login
- Moved css into a separate file
- Applied a number of css workarounds to make the page look OK

I tested that pagination, Agree button, Withdraw button and small page sizes work OK.

Here's how it looks
image

### CODE REVIEW

File: app/assets/stylesheets/identifications_by_login.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+.bootstrap .readable {font-size:125%;}
+.bootstrap .observations.mini .icon {top:8px;}
+.bootstrap .needs_id_notice { 
+  background-color: #FFEE91;
+  color: #857430;
+}
+.bootstrap .research_grade_notice { 
+  background-color: #DCEAA3;
+  color: #529214;
+}
```
Comment (seanclifford) [inline app/assets/stylesheets/identifications_by_login.css:34]: These are the extra css workarounds/hacks I added.

File: app/views/identifications/_identification_with_observation.html.erb [hunk 2]
```diff
@@ -1,19 +0,0 @@
-<%- identification ||= identification_with_observation || @identification -%>
-<div class="mini <%= 'compact' if request.format.mobile? %> observations observationcol column span-8">
```
Comment (seanclifford) [inline app/views/identifications/_identification_with_observation.html.erb:2]: I left out the compact style in the haml. I'm guessed that it's not required with bootstrap columns resizing?

File: app/assets/stylesheets/identifications_by_login.css (hunk 1 above)
Comment (seanclifford) [inline app/assets/stylesheets/identifications_by_login.css:34]: These are the extra css workarounds/hacks I added.

**Review by kueda — CHANGES_REQUESTED** [review]
In addition to the comments, two requests:

- [ ] Move `app/assets/stylesheets/identifications_by_login.css` to `app/assets/stylesheets/identifications/by_login.scss`
- [ ] Make sure all files end with a newline

Also, not something you need to do here, but in general I prefer to `git mv` files when converting from ERB to HAML, as opposed to deleting the ERB versions and adding the HAML ones. Helps preserve the history.

File: app/assets/stylesheets/identifications_by_login.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  text-decoration: underline;  
+  font-size: smaller;  
+  cursor: pointer; 
+  padding:0; 
+  border:0; 
+  float:none; 
+  background-color: transparent;
+}
+
+.bootstrap .readable {font-size:125%;}
```
[the remark this answers is not in the document]
Comment (seanclifford) [reply app/assets/stylesheets/identifications_by_login.css]: Removed. 
I'm not sure which alignment you're referring to though - I didn't spot an improvement after it was gone.

**Comment by seanclifford** [comment]
Thank you for the review @kueda!
I think I've made all requested changes now.
On the `git mv` front - I'm happy to redo the commits and create a move commit before applying the haml changes if you'd like the history preserved. I'd have to recreate the branch and PR I imagine though.

File: app/assets/stylesheets/identifications_by_login.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+.bootstrap .observations.mini .icon {top:8px;}
+.bootstrap .needs_id_notice { 
+  background-color: #FFEE91;
+  color: #857430;
+}
+.bootstrap .research_grade_notice { 
+  background-color: #DCEAA3;
```
[the remark this answers is not in the document]
Comment (seanclifford) [reply app/assets/stylesheets/identifications_by_login.css]: I went with the white on green. I think it's an improvement!

File: app/assets/stylesheets/identifications/by_login.scss [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  margin-bottom: 0;
+  border: 0;
+}
+
+.identification {
+  padding: 10px;
+  background-color: #eee;
+
+  .identification_image { width: 75px; }
+  .identification_body { float: left; }
```
Comment (kueda) [inline app/assets/stylesheets/identifications/by_login.scss:14]: Sorry I missed this on the first pass: remove this and add `display: flex` to `.identification` to achieve the same effect while not wrapping at smaller window sizes.

**Review by kueda — CHANGES_REQUESTED** [review]
Sorry, just that one more thing. No need to redo things with `git mv`, just something to keep in mind for similar work in the future.

File: app/assets/stylesheets/identifications/by_login.scss (hunk 5 above)
Comment (seanclifford) [reply app/assets/stylesheets/identifications/by_login.scss]: Fixed, thanks. 
I put `display: flex` under the bootstrap section, as the bootstrap_bundle css of `display: block` seemed to be overriding it.

**Pull request merged**